Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/ThemeInspector.cs

```diff
@@ -1228,54 +1219,53 @@ public static void RenderValue(SerializedProperty item, string name, string prop
             }
         }
 
-        public static void RenderThemeStates(SerializedProperty settings, State[] states, int margin)
+        public static void RenderThemeStates(SerializedProperty settings, State[] states, int margin = 0)
         {
             GUIStyle box = InspectorUIUtility.Box(margin);
 
             EditorGUILayout.BeginVertical(box);
-            for (int n = 0; n < states.Length; n++)
-            {
-                InspectorUIUtility.DrawLabel(states[n].Name, 12, InspectorUIUtility.ColorTint50);
-
-                EditorGUI.indentLevel = indentOnSectionStart + 1;
 
-                for (int j = 0; j < settings.arraySize; j++)
+                for (int n = 0; n < states.Length; n++)
                 {
-                    SerializedProperty settingsItem = settings.GetArrayElementAtIndex(j);
+                    InspectorUIUtility.DrawLabel(states[n].Name, 12, InspectorUIUtility.ColorTint50);
```

Comment:
Why 12?

Reply:
I don't know, that's actually original code. @killerantz ?

Reply:
That's the font size of the label, it needs to be a little larger for eye flow. I could grab the default size of label and add to that. 

GUIStyle labelStyle = new GUIStyle(EditorStyles.boldLabel);
int labelSize = labelStyle.fontSize + 2; (assuming labelStyle.fontSize is 10)

or 
InspectorUIUtility.DefaultFontSize + 2.

Reply:
Maybe InspectorUIUtility.DefaultFontSize * ThemeStateFontScale (which equals 1.2)?

Reply:
@Troy will you be making this change? If yes please just run the PR and notify me when I should pay attention so I can merge this in.

Reply:
Yes I can make the change. I would like @killerantz approval for the PR since it is my understanding he is the core author for all of this.